timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -47,8 +73,44 @@ services:
           namespace: string
           request: ConjureStartTransactionsRequest
         returns: ConjureStartTransactionsResponse
+      startTransactionsWithWatches:
+        http: POST /stww/{namespace}
+        args:
+          namespace: string
+          request: StartTransactionsWithWatchesRequest
+        returns: StartTransactionsWithWatchesResponse
+        docs: |
+          The returned lock watch state the state taken atomically along with issuing the start timestamps. A failed
+          update means the lock watch state changed too much while start timestamps were being issued, which is
+          indicative of poorly registered lock watches.
```

Comment:
This is reasonable, but is there a reason we don't want to or can't use existing `startTransactions`? This could add some overhead if clients aren't using the watches, but I imagine that will be changing soon.

Reply:
Yes, there should not need to be any new endpoint, here.